src/Sulu/Bundle/AdminBundle/Resources/js/components/Breadcrumb/README.md

```diff
@@ -0,0 +1,16 @@
+The `Breadcrumb` indicates the current page’s location within a navigational hierarchy.
+
+```
+const handleClick1 = () => {
+    alert(`You clicked on crumb 1`);
+};
+const handleClick2 = (value) => {
+    alert(`You clicked on crumb ${value}`);
+};
+
+<Breadcrumb>
+    <Breadcrumb.Crumb onClick={handleClick1}>Root Directory</Breadcrumb.Crumb>
+    <Breadcrumb.Crumb value={2} onClick={handleClick2}>...</Breadcrumb.Crumb>
```

Comment:
I think it is debatable if we should have the possibility of different `onClick` handlers for each breadcrumb. Shouldn't there be only one on the container component, which is always called with the passed value?

Reply:
Would also be more consistent with the behavior of the `Tabs`, wouldn't it? And I don't think that having completely different click handlers for each breadcrumb is something we want to support anyway (?) /cc @chirimoya